### PR TITLE
chore: fix some typos in comments

### PR DESCRIPTION
**Pull request by goodfirm**
#### Type of change

- Bug fix
- New feature
- Improvement (improvement to code, performance, etc)
- Test update
- Documentation update

#### Description

fix some typos in comments

### CODE REVIEW

**Review by ale-linux — DISMISSED** [review]

File: core/ledger/kvledger/state_listener_test.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ func TestStateListener(t *testing.T) {
 	require.NoError(t, sim2.SetState(namespace, "key3", []byte("value3")))
 	sim2.Done()
 
-	// Simulate tx3 - this neighter conflicts with tx1 nor with tx2
+	// Simulate tx3 - this nighter conflicts with tx1 nor with tx2
```
Comment (satota2) [inline core/ledger/kvledger/state_listener_test.go]: @goodfirm It seems like 'neither' is the correct word, not 'nighter'.
```suggestion
	// Simulate tx3 - this neither conflicts with tx1 nor with tx2
```

**Review by satota2 — CHANGES_REQUESTED** [review]
@goodfirm I have found some minor corrections. Please check it.

File: core/ledger/kvledger/state_listener_test.go (hunk 1 above)
Comment (goodfirm) [reply core/ledger/kvledger/state_listener_test.go]: Thanks! Modified

Please review again

**Comment by satota2** [comment]
@goodfirm Thank you for the update. It seems there is still a typo in the revised version: "neighter" should be corrected to "neither." Could I kindly ask you to make this correction once more?

**Comment by goodfirm** [comment]
> @goodfirm Thank you for the update. It seems there is still a typo in the revised version: "neighter" should be corrected to "neither." Could I kindly ask you to make this correction once more?

Of course, thank you for your comment.

Modified

**Review by satota2 — APPROVED** [review]
@goodfirm  Thank you for your updates. LGTM!

**Pull request merged**